docs/api.md

```diff
@@ -43,7 +43,7 @@
   * [event: 'requestfinished'](#event-requestfinished)
   * [event: 'response'](#event-response)
   * [page.$(selector)](#pageselector)
-  * [page.$$(selector)](#pageselector)
+  * [page.$$(selector)](#pageselector-1)
```

Comment:
unfortunately, the table-of-contents is generated, and there's a bug in a generator we use. 

So manual changes won't work.